File: Java/Sorting.java
import java.util.Scanner;

public class Sorting
{
    public static void arrays(int arr[], int size)
    {
        int aux;
        for(int i = 0; i < size - 1; i++)
        {
            for(int j = 0; j < size - i - 1; j++)
            {
                if(arr[j] > arr[j + 1])
                {
                    aux = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = aux;
                }
            }
        }
        System.out.print("Sorted array: ");
        for(int i = 0; i < size; i++)
        {
            System.out.print(arr[i] + " ");
        }
        System.out.println();
    }

    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Type the array size: ");
        int size = scanner.nextInt();
        int array[] = new int[size];
        System.out.println("Type the array values:");
        for(int j = 0; j < size; j++)
        {
            System.out.print("Value " + (j + 1) + ": ");
            array[j] = scanner.nextInt();
        }
        arrays(array, size);
    }
}

File: CSharp/Sorting.cs
using System;

public class Progrma
{
    public static void sorting(int[] array, int size)
    {
        int aux;
        for(int i = 0; i < size - 1; i++)
        {
            for(int j = 0; j < size - i - 1; j++)
            {
                if(array[j] > array[j + 1])
                {
                    aux = array[j];
                    array[j] = array[j + 1];
                    array[j + 1] = aux;
                }
            }
        }
        Console.Write("Sorted array: ");
        for(int i = 0; i < size; i++)
        {
            Console.Write(array[i] + " ");
        }
    }

    public static void Main()
    {
        char option;
        do
        {
            int size;
            Console.Write("Type the array size: ");
            size = Convert.ToInt32(Console.ReadLine());
            int[] array = new int[size];
            Console.WriteLine("Type the array values:");
            for(int k = 0; k < size; k++)
            {
                Console.Write("Value " + (k + 1) + ": ");
                array[k] = Convert.ToInt32(Console.ReadLine());
            }
            sorting(array, size);
            Console.Write("\nType r to return, or e to exit: ");
            option = Convert.ToChar(Console.ReadLine());
        } while(option == 'r');
    }
}